src/cryptoadvance/specter/static/styles.css

```diff
@@ -1019,6 +1019,18 @@ nav.collapse-on-mobile a.mobile-nav-icon {
     display: none;
 }
 
+
+/* The viewable height is often smaller than the height given by the browser. 
+See: https://css-tricks.com/the-trick-to-viewport-units-on-mobile/
+This style is used for a div at the bottom in the <main> element to create a distance at the bottom of the pages
+*/
+.mobile-distance-element-botom {
```

Comment:
Please fix the spelling of bottom

Reply:
Sorry, I didn't even see the typo until now!   Done.